src/variables.jl

```diff
@@ -54,11 +51,24 @@ end
 function keywordify(kw::Expr)
     (kw.args[1], esc_nonconstant(kw.args[2]))
 end
-function VariableInfo(; lowerbound=NaN, upperbound=NaN, start=NaN, binary=false, integer=false)
-    VariableInfo(lowerbound !== NaN, lowerbound, upperbound !== NaN, upperbound, false, NaN, start !== NaN, start, binary, integer)
+function VariableInfoExpr(; lowerbound=NaN, upperbound=NaN, start=NaN, binary=false, integer=false)
+    VariableInfoExpr(lowerbound !== NaN, lowerbound, upperbound !== NaN, upperbound, false, NaN, start !== NaN, start, binary, integer)
```

Comment:
Use `!isnan` instead of `!== NaN`.

Reply:
`isnan(::Expr)` is not defined, I have added a comment